### ISSUE TITLE
We should have a Dev branch

### ISSUE THREAD

**Issue by Frantz-B** [tesla809/Pongzi#15]
It's important to have a Dev branch so that people can push their code/changes to this branch first.  Then it can be pushed into Master.

This helps master to always be clean

**Comment by tesla809** [comment]
I agree @Frantz-B. The more collaborators we have, the more risk there could be for a break in the code. So, would you like to create a master branch, then update the README.md with a COLLABORATORS section that mentions the work flow?
